lib/transformer/graph_methods.rb

```diff
@@ -0,0 +1,42 @@
+module Transformer
+  module GraphMethods
+    def self.all
```

Comment:
@antw this is where I initialize all the graph methods as attributes. 

I do this by turning their keys into something like:

`#{ node_key_#{ method }`

So for instance `households_final_demand_for_electricty` which has for example `[demand, output]` as graph_methods, would turn into... :

```
households_final_demand_for_electricty_demand
households_final_demand_for_electricty_output
```

... in the front-end of ETLocal. 

_Note: I am aware that `output` itself as a single attribute for a form is not sufficient, but provided by Michiel that it's not going to be used in the front-end yet, I decided to leave it as such for now._

I also give each node and edge an export hash, this is useable in the conversion from the ETLocal form attributes to the actual `graph_values.yml`.

Does this seem like an okay approach?